lldb/test/API/functionalities/mtc/swift-property/TestMTCSwiftProperty.py

```diff
@@ -41,6 +41,7 @@ def mtc_tests(self):
 
         self.expect(
             "thread info -s",
+            ordered=False,
```

Comment:
What makes the output nondeterministic? Is it not sufficient to reorder the substrings?

Reply:
This is a JSON blob, so technically a dictionary.